src/server/native_controllers/native_controller.js

```diff
@@ -0,0 +1,41 @@
+import { Observable, ReplaySubject } from 'rxjs';
+
+export class NativeController {
+
+  killed = false;
+
+  constructor(pluginId, process) {
+    this.pluginId = pluginId;
+    this.process = process;
+
+    this._ready$ = new ReplaySubject(1);
+    Observable
+      .fromEvent(process, 'message')
+      .filter(message => message === 'ready')
+      .first()
+      .subscribe(this._ready$);
+
+    this._started$ = Observable
```

Comment:
Can we move this observable definition into the `start()` function? Since it is defined next to the Observable that populates `this._ready$` it seemed it would have a similar life cycle, but it's really only useful within the `start()` function and its purpose is clear when it's next to `this.process.send('start')`.